### ISSUE TITLE
Purified Ores deleted

### ISSUE THREAD

**Issue by nathan67003** [theflogat/Technomancy#193]
Everything was working fine in my custom modpack but when I added GregTech, the purified ores for Osmium, Indium, Dark Iron, Cadmium and Gold from Technomancy are said to not exist anymore. Here's the end of the launcher's integrated log, the prompt substitute starts at about line 893. http://paste.ubuntu.com/25019924/

Here's also the _entire_ FML client log.

http://paste.ubuntu.com/25024218/ http://paste.ubuntu.com/25024221/ http://paste.ubuntu.com/25024226/ http://paste.ubuntu.com/25024232/

**Comment by nathan67003** [comment]
I'll try fixing it via config by changing the assigned names.

**Comment by nathan67003** [comment]
Editing the configs (switching pureOsmium to purifiedOsmium, for instance, as well as changing the names to things like Osmiumm) didn't work.

**Comment by Mordenkainen** [comment]
If I get a chance I will look, but historically GT has been bad for other mods, and if it is a clash with that it probably won't be fixed. I don't feel like fighting my way through all Gregs nerfs and hacks.

**Comment by Mordenkainen** [comment]
OK. I am going to need some more info.

With GT installed, check each of the "missing" purified ore types for the following:
1. Does each have an ore and a matching ingot?
2. Do the ore dictionary names for these items match up? I.E. "oreOsmium" & "ingotOsmium"? You can see these via the debug options in NEI, I forget the exact option name.
3. Is there a furnace smelting recipe that goes from ore -> ingot?

If any of the above are not true, the purified ore will not be created.

If I were to take a guess, I would say that GT probably removed the furnace recipes for the ingots and replaced them with something that forces you to use some GT mechanic instead.

**Comment by nathan67003** [comment]
1. Yes. The ores and ingots are independent from GT.
2. Yes, as the items not in GT are unaffected by its installation.
3. Aaah, that's very probably what happened. I looked around NEI and asked Greg and apparently, most furnace recipes regarding metals get wrecked due to a core mechanic of the mod. I guess it could be called a non-issue, then. Probably also the reason why Cobalt, Iridium added by AOBD and Ardite don't register.

Closing issue! Sorry for the trouble.
